pykern/pkunit.py

```diff
@@ -184,10 +184,14 @@ def file_eq(expect_path, *args, **kwargs):
     """
     import pykern.pkjson
     import pykern.pkconst
+    from pykern.pkdebug import pkdp
 
     def _convert_xlsx(expect_path, actual_path):
         if expect_path.ext == '.csv' and not actual_path.check(file=True):
-            _xlsx_to_csv(expect_path, actual_path)
+            _xlsx_to_csv(pkcollections.PKDict(
```

Comment:
We allow unqualified PKDict. In emacs, you say "C-c t" which adds this:

```py
# -*- coding: utf-8 -*-
u"""?

:copyright: Copyright (c) 2022 RadiaSoft LLC.  All Rights Reserved.
:license: http://www.apache.org/licenses/LICENSE-2.0.html
"""
from pykern.pkcollections import PKDict
from pykern.pkdebug import pkdc, pkdlog, pkdp
```

You should just insert that header into files